### DISCUSSION TITLE
Specifying a note with local AI chat model causes empty requests

### DISCUSSION THREAD

**Discussion by dragos240** [brianpetro/obsidian-smart-connections#685] [General]
Hello,

When I specify a note with the wiki-link syntax, I notice it begins sending empty responses to my server. Everything works before I specify a page, but if I specify a note, it breaks and even requests NOT specifying a note will start generating empty requests. I know this because I have the `---verbose` flag turned on in text-generation-webui. Before I specify a note, I can see the request with a `## instruction\n\nINPUT TEXT HERE` in the terminal running the local server. But after I specify a note, I see no user specific input in the requests, which generates nonsense replies in the chat.

I've tried completely deleting `.smart-connections` in the main vault folder and also deleting the plugin in `plugins`, but even if I reconfigure everything again, the behavior persists. It's using my local AI server, but it's just sending empty responses to it. I'm not sure how to fix it

**Comment by dragos240** [comment]
Example:
![image](https://github.com/brianpetro/obsidian-smart-connections/assets/704081/65657107-d5d3-48f6-94fa-e1d92d125661)

This has no relation to the referenced note

**Comment by dragos240** [comment]
Another attempt:
![image](https://github.com/brianpetro/obsidian-smart-connections/assets/704081/9a046d00-d320-4c56-b68b-a69efcabfb96)

**Comment by brianpetro** [comment]
Hi @dragos240 

Thanks for the detailed feedback. 

I moved this to the discussions section because it's likely that this is a model-specific issue rather than a bug that can be fixed in Smart Connections.

Which local model was used to generate those responses? 🌴
